plugins/modules/nd_compliance_requirement_config_manual.py

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, Akini Ross (@akinross) <akinross@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: nd_compliance_requirement_config_manual
+version_added: "0.2.1"
+short_description: Manage manual configuration type compliance requirements
+description:
+- Manage manual configuration type on Cisco Nexus Dashboard Insights (NDI).
+author:
+- Akini Ross (@akinross)
+options:
+  insights_group:
+    description:
+    - The name of the insights group.
+    type: str
+    required: yes
+    aliases: [ fab_name, ig_name ]
+  name:
+    description:
+    - The name of the compliance requirement.
+    type: str
+  description:
+    description:
+    - The description of the compliance requirement.
+    type: str
+    aliases: [ descr ]
+  enabled:
+    description:
+    - Enable the compliance requirement.
+    type: bool
+  sites:
+    description:
+    - Names of the Assurance Entities.
+    type: list
+    elements: str
+  object:
+    description:
+    - Container for all matching criteria attached to the object.
+    type: dict
+    suboptions:
+      type:
+        description:
+        - The object type of the object.
+        type: str
+        required: yes
+        choices: [ tenant, vrf, bd, epg, contract, subject, filter ]
+      includes:
+        description:
+        - Container for all matching criteria to include.
+        type: list
+        required: yes
+        elements: dict
+        suboptions:
+          type:
+            description:
+            - The object type of the match.
+            type: str
+            required: yes
+            choices: [ tenant, vrf, bd, epg, ap, l3out, l3instp, l2out, l2instp, filter, subject, contract ]
```

Comment:
are the external epgs (l2instp and l3instp) referenced as External EPGs or instps in the UI?

Reply:
in ui